sdk/aqueduct/api_client.py

```diff
@@ -371,16 +371,19 @@ def _construct_register_workflow_request(
     def refresh_workflow(
         self,
         flow_id: str,
-        serialized_params: Optional[str] = None,
+        serialized_param_specs: Dict[str, str],
     ) -> None:
+        """
+        `serialized_param_specs`: a dictionary from parameter names to serialized
+            parameter specs, which include the base64-encoded parameter value and the
+            parameter serialization type.
+        """
         headers = self._generate_auth_headers()
         url = self.construct_full_url(self.REFRESH_WORKFLOW_ROUTE_TEMPLATE % flow_id)
 
-        body = {}
-        if serialized_params is not None:
-            body["parameters"] = serialized_params
+        body = {"parameters": serialized_param_specs}
```

Comment:
The format of the body is - a dictionary from parameter name to json-serialized parameter spec. This parameter spec includes a base64-encoded & serialized parameter value, along with the serialization type.

Because of this, the backend needs to unwrap twice to get all the information - once to get the parameter spec, and once to get the parameter value in the spec.